client/extensions/woocommerce/components/form-dimensions-input/style.scss

```diff
@@ -1,6 +1,17 @@
 .form-dimensions-input {
 	display: flex;
 	justify-content: flex-start;
+	flex-wrap: nowrap;
+
+	.form-text-input-with-affixes {
+		flex-grow: 2;
+		flex-direction: row;
+	}
+
+	.form-dimensions-input__length,
+	.form-dimensions-input__width {
+		border-right: 0;
```

Comment:
@kellychoffman Saw you switched to this rather than using the negative margin I had originally. The issue with border removal however is the focus display;

![missing-border](https://cldup.com/CnIKMhvXW3-3000x3000.png)

That missing border-right bothers my ocd :p

Removing the border does feel more appropriate code-wise, but are there any major concerns with a 1px negative margin/transform?

If we _do_ go with the border removal method, [these styles](https://github.com/Automattic/wp-calypso/blob/146e3df1714de3a82880c52e015d9a03e7bed732/client/extensions/woocommerce/components/form-dimensions-input/style.scss#L16-L29 ) can be removed. They're there to ensure the focused input appears 'on top' of the others visually and that the borders all match, when using a negative margin.